### ISSUE TITLE
[UI/UX] Want to add the HamBurger during the small size of device

### ISSUE THREAD

**Issue by Bhumika-00** [harshithtunuguntla/project-osus#28]
## Description
Want to add the HamBurger during the small size of device

## Proposed Changes
This issue is requested by the team 
## Examples
If possible, provide mockups or references to similar designs that illustrate your suggestions.

**Comment by Deepak-Perla** [comment]
Hi Bhoomika,

Thankyou for pointing it out. You can make then buttons hidden under the hamburger icon. And it turns into close button when opened. PFA image for you reference. Follow the theme of the app, and make sure you pull the latest changes before starting to code.

![image](https://github.com/user-attachments/assets/9926ac93-1739-423e-a956-fab071eeede0)

**Comment by Bhumika-00** [comment]
Okay please assign me this issue

**Comment by Bhumika-00** [comment]
I have made the changes please give the access and review it
  VID_20241012_155130_0_COMPRESSED.mp4
<https://drive.google.com/file/d/1KEvt7ar52edWHH_dUbhcUu7o9desv4kl/view?usp=drivesdk>

**Comment by Deepak-Perla** [comment]
The hamburger icon looks cool, but 
- How does it perform when the user clicked the hamburger icon? 
- And how does the menu bar look like in mobile version?

**Comment by Bhumika-00** [comment]
When the user will click on hamburger icon the document and know about
section will be available

**Comment by Deepak-Perla** [comment]
Could you share any image respective to that.

**Comment by Bhumika-00** [comment]
I don't know why hamburger is not working let me do something else for it

**Comment by DCGUY009** [comment]
Hi @Bhumika-00 closing this issue as there is another issue open w.r.t this. If you want to work on this, you can show your interest in this issue number #51
